### PR TITLE
improve indentation of step-36 and minor cleanup

### PR DESCRIPTION
**Pull request by davydden**
previous indentation https://github.com/dealii/dealii/blob/dealii-9.0/examples/step-36/step-36.cc#L268-L285

@tamiko i could not win here 😞

### CODE REVIEW

File: examples/step-36/step-36.cc [hunk 1]
```diff
@@ -276,17 +271,21 @@ namespace Step36
...
-                     fe_values.shape_value(i, q_point) *
-                     fe_values.shape_value(j, q_point)) *
-                  fe_values.JxW(q_point);
-
-                cell_mass_matrix(i, j) += (fe_values.shape_value(i, q_point) *
-                                           fe_values.shape_value(j, q_point)) *
-                                          fe_values.JxW(q_point);
+                cell_stiffness_matrix(i, j) +=           //
+                  (fe_values.shape_grad(i, q_point) *    //
+                     fe_values.shape_grad(j, q_point)    //
```
Comment (drwells) [inline examples/step-36/step-36.cc]: I am a bit surprised that clang-format wants to add two spaces here but not in the mass matrix. Would surrounding the contraction with parentheses fix it?

I am not overly concerned, just slightly annoyed at clang-format's apparent inconsistency (which I suppose is due to the `+` sign on the next line).

Comment (tamiko) [reply examples/step-36/step-36.cc]: Yes it is due to the + sign.

File: examples/step-36/step-36.cc [hunk 2]
```diff
@@ -472,51 +471,19 @@ namespace Step36
...
-                << "----------------------------------------------------"
-                << std::endl;
-      std::cerr << "Unknown exception!" << std::endl
-                << "Aborting!" << std::endl
-                << "----------------------------------------------------"
-                << std::endl;
-      return 1;
-    }
+  EigenvalueProblem<2> problem("step-36.prm");
+  problem.run();
```
Comment (drwells) [inline examples/step-36/step-36.cc]: One more thing: from the discussion in #6784 we should keep this so that MSVC can produce nice output.

Comment (davydden) [reply examples/step-36/step-36.cc]: ah well, that's a bummer. Personally I would consider it MSVC issue and get rid of those lines of code throughout. But I will put them back in here.

**Pull request merged**